Configuration/TSconfig/ContentWizard/_wizard.ts

```diff
@@ -0,0 +1,16 @@
+# add multicolumn container to common tab
+mod {
```

Comment:
Please prevent unnecessary brackets.
mod.wizards.newContentElement.wizardItems.common.elements should be the first line